samcli/lib/remote_invoke/lambda_invoke_executors.py

```diff
@@ -0,0 +1,147 @@
+"""
+Remote invoke executor implementation for Lambda
+"""
+import base64
+import json
+import logging
+from json import JSONDecodeError
+from typing import Any, Dict, cast
+
+from botocore.exceptions import ClientError, ParamValidationError
```

Comment:
One thing about any boto imports is that they add latency, so be mindful about that as its a large lib.